subprojects/gradle/critical-path/api/src/main/kotlin/com/avito/android/critical_path/CriticalPathRegistry.kt

```diff
@@ -0,0 +1,32 @@
+package com.avito.android.critical_path
+
+import com.avito.android.critical_path.internal.PathBuildProvider
+import com.avito.android.gradle.metric.GradleCollector
+import com.avito.kotlin.dsl.isRoot
+import org.gradle.api.Project
+
+public object CriticalPathRegistry {
```

Comment:
I want to see a comment with motivation `Why this class is exist`

Reply:
Added in #1057